packages/react/presence/src/Presence.tsx

```diff
@@ -143,5 +143,29 @@ function getAnimationName(styles?: CSSStyleDeclaration) {
   return styles?.animationName || 'none';
 }
 
+// Before React 19 accessing `element.props.ref` will throw a warning and suggest using `element.ref`
+// After React 19 accessing `element.ref` does the opposite.
+// https://github.com/facebook/react/pull/28348
+//
+// Access the ref using the method that doesn't yield a warning.
+function getElementRef(element: React.ReactElement) {
```

Comment:
🤯

Reply:
Would have to be a separate package—I'd say we can keep it like this for now and extract it later if there's 3+ uses. I think two copy-pastes is OK for now.